nengo_spa/actions.py

```diff
@@ -26,10 +30,15 @@ class Parser(object):
         'reinterpret': Reinterpret,
         'translate': Translate}
 
-    def __init__(self, vocabs=None):
-        if vocabs is None:
-            vocabs = {}
-        self.vocabs = vocabs
+    def __init__(self, locals_=None):
```

Comment:
Nitpick: would it be better to name this `local_vars` or something else? I understand it's called `locals_` because `locals` is a reserved keyword, but a trailing underscore feels a bit weird?